spec.html

```diff
@@ -39214,7 +39214,7 @@ <h1>Properties of the AsyncFunction Constructor</h1>
 
       <p>The value of the [[Extensible]] internal slot of the AsyncFunction constructor is *true*.</p>
 
-      <p>The value of the *name* property of the AsyncFunction constructor is "AsyncFunction".</p>
+      <p>The value of the `name` property of the AsyncFunction constructor is "AsyncFunction".</p>
```

Comment:
Should the string have backticks also, since it's a JS value?

Reply:
It should have something. It's arguable whether that should be backticks or asterisks, but backticks almost always wins the argument. I'll add a commit that inserts backticks.